apps/github-cascading-app/README.md

```diff
@@ -0,0 +1,63 @@
+# Github Cascading App
+
+A GitHub App to execute cascading between the repository branches.
+
+## How to install
+
+The Otter Cascading application can be installed to your organization following via [this link](https://github.com/apps/otter-cascading).
+
+The application will need the following permission to work:
+
+- **Read** access to checks and metadata
+- **Read** and **write** access to code and pull requests
+
+## How it works
+
+### What it does
+
+The application will look for branches matching the pattern defined in the `cascadingBranchesPattern` option.
+Then it will sort the branches, and based on the current branch you gave as input will figure out if the cascading is needed and on which branch.
```

Comment:
```suggestion
Then it will sort the branches and, based on the current branch you gave as input, will figure out if the cascading is needed and on which branch.
```